app/controllers/api/v1/users_controller.rb

```diff
@@ -1,18 +1,18 @@
 class Api::V1::UsersController < ApplicationController
-def create
-  user = User.create_account(user_params)
-  if user.save
-    render json: UserSerializer.new(user), status: :created
-  else
-    error = user.errors.full_messages.to_sentence
-    status = error.include?('taken') ? :conflict : :forbidden
-    render json: { error: error }, status: status
+  def create
+    user = User.create_account(user_params)
+    if user.save
+      render json: UserSerializer.new(user), status: :created
+    else
+      error = user.errors.full_messages.to_sentence
+      status = error.include?('taken') ? :conflict : :forbidden
```

Comment:
Note to self: if the `error` include the word `taken` (as in 'Email has already been taken') then the conflict is forbidden and a 409 error is rendered